src/components/card/Card.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../styles/variables';
+
+.bdl-Card {
+    background-color: $white;
+    border-radius: 3px;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
+    margin: 0 auto;
+    max-width: 900px;
```

Comment:
What is the reason for selecting this value for max width? It seems tailored to a specific use-case, which isn't ideal for a reusable component.